EIPS/eip-draft_multi_byte_opcodes.md

```diff
@@ -0,0 +1,37 @@
+---
+eip: 4520
+title: Mult-byte opcodes prefixed by EB and EC.
+description: Future opcodes may be introduced at 
```

Comment:
The description should be a complete and standalone sentence.  In this case, you could probably just use the abstract's contents here since the abstract is so short.